pykafka/test/utils.py

```diff
@@ -2,6 +2,16 @@
 
 from pykafka.test.kafka_instance import KafkaInstance, KafkaConnection
 
+try:
+    import unittest2 as unittest
```

Comment:
This may not even be needed, given that we only support python 2.7 and up. Would save us a pip install if we just used the standard lib.